_topic_map.yml

```diff
@@ -110,6 +110,8 @@ Topics:
     File: installing-gcp-default
   - Name: Installing a cluster on GCP with customizations
     File: installing-gcp-customizations
+  - Name: Restricted network GCP installation
+    File: installing-restricted-networks-gcp
```

Comment:
So why this section was moved to IPI-on-GCP, from my understanding, it should be placed where it was - `Installing in restricted networks `

Reply:
I'm working on a change to reorder the installation table of contents based on cloud provider. I that change to merge before this one does.